scripts/templates/field_details.j2

```diff
@@ -136,6 +136,11 @@ Note also that the `{{ fieldset['name'] }}` fields are not expected to be used d
 | {{ entry['short'] }}
 {%- endif %}
 
+{% if entry['normalize'] and 'array' in entry['normalize'] -%}
+
+Note: this reuse should contain an array of <fieldset> objects.
+
+{% endif %}
```

Comment:
```suggestion
{% endif -%}
```

Nit: Jinja2 whitespace can be obnoxious. If you want to drop the extra blank lines now showing up in `field-details.asciidoc`, you can adjust the whitespace trimming in the template.

It's also OK with the extra lines, though; it shouldn't change how it renders.

Reply:
Thank you for this! I was trying to figure this out for so long and couldn't